--- dropwatch-1.5.3/src/Makefile.am.orig	2020-03-19 14:19:40.000000000 +0100
+++ dropwatch-1.5.3/src/Makefile.am	2021-07-21 01:18:23.495547294 +0200
@@ -1,7 +1,7 @@
 
 bin_PROGRAMS = dropwatch dwdump
 
-AM_CFLAGS = -g -Wall -Werror $(LIBNL3_CFLAGS) $(READLINE_CFLAGS)
+AM_CFLAGS = -g -Wall $(LIBNL3_CFLAGS) $(READLINE_CFLAGS)
 AM_LDFLAGS = $(LIBNL3_LIBS) -lnl-genl-3 $(READLINE_LIBS) -lpcap
 AM_CPPFLAGS = -D_GNU_SOURCE
 
